Directory.Build.targets

```diff
@@ -1,4 +1,7 @@
 <!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
 <Project>
   <Import Project="Sdk.targets" Sdk="Microsoft.DotNet.Arcade.Sdk" />
+  <ItemGroup>
+    <PackageReference Include="System.Security.Cryptography.X509Certificates" Version="4.3.0" />
```

Comment:
We could upgrade this repo to use centralized package versioning and then force the version this way. 

That would take a bit of work. I've marked it as `PrivateAssets="all"` so it's not included as a explicit dependency in our nuget packages.

Reply:
@jmarolf OK, then I'm going to make a similar request as I made to @sharwell: add a comment here explaining why this is here, as otherwise it's a real headscratcher.

Reply:
added a comment explaining the situation.